tests/template-gen.service.test.ts

```
@@ -0,0 +1,1191 @@
+  });
+
+  // -----------------------------------------------------------------------
+  // SYSTEM_PROMPT is forwarded verbatim as the system message
+  // -----------------------------------------------------------------------
+  test("VAL-M3-PROMPT-006: system prompt forwarded verbatim to OpenRouter", async () => {
+    const mod = await import("@/api/v2/agent-templates/services/templateGen");
+    generateTemplate = mod.generateTemplate;
+
+    const { SYSTEM_PROMPT } = await import(
+      "@/api/v2/agent-templates/lib/system-prompt"
+    );
```

ESLint

[error] Replace `·await·import(⏎······"@/api/v2/agent-templates/lib/system-prompt"⏎····` with `⏎······await·import("@/api/v2/agent-templates/lib/system-prompt"`

(prettier/prettier)